.gitignore

```diff
@@ -0,0 +1,41 @@
+# Copyright: Chetna (cheatiiit) https://github.com/cheatiiit
```

Comment:
Have you discussed this with the PMs or @rosariorobinson ? I think the copyright should probably go to the Systers PowerUp copyright